### PR TITLE
Introducing the tickets blog post

### PR DESCRIPTION
**Pull request by benwerd**
# Description

A blog post describing the new tickets app.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

screencapture-localhost-3002-blog-introducing-tickets-2019-04-30-12_12_25

### CODE REVIEW

File: unlock-protocol.com/blog/introducing-tickets.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+like [Metamask](https://metamask.io) to purchase a ticket at a nominal price.
+
+## How tickets work with Unlock
+
+Every Unlock transaction - whether for content, an event, software licenses, or anything else - works by selling a _key_ 
+to a _lock_ using Ethereum. Tickets are no different.
+
+![Unlock ticket creation screen](/static/images/blog/introducing-tickets/create-an-event.png)
+
+Once you've created a lock, you'll have access to a number of ways to use it: different applications that sit on top of
```
Comment (julien51) [inline unlock-protocol.com/blog/introducing-tickets.md]: I would use "creator" to be more specific that "you".
For example:
> As a meetup organizer, after having created a lock, you can use it in different ways...

File: unlock-protocol.com/blog/introducing-tickets.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+the Unlock Protocol. The ticketed event application allows you to save a date, title, description, and location for the
+event. These aren't stored on the blockchain itself: instead, they create a database record that references the lock's 
+Ethereum address.
+
+A user with an Ethereum wallet like Metamask can buy a key with one click. The event page then shows a QR code, which
+contains the user's public key and a signed version of the lock address. This can be verified at the door with a
+commodity QR code reader, which are now built into every major mobile operating system.
```
Comment (julien51) [inline unlock-protocol.com/blog/introducing-tickets.md]: > which are now built into camera applications for every major mobile operating system.

**Comment by julien51** [comment]
@benwerd Let's revisit this one?

**Comment by benwerd** [comment]
I rewrote this for August. PTAL.


screencapture-localhost-3002-blog-introducing-tickets-2019-08-05-11_07_00

**Comment by julien51** [comment]
Hey, I think it would be really cool if we showed how the QR code which checks the validity of the ticket works... do you think you could record that? If not I'll do it with @cnasc tomorrow.

**Pull request merged**